src/components/views/adminView/Cities.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import { TextField, Input, CircularProgress } from '@material-ui/core'
+import { fetchData } from '../../../utils'
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+
+export const Cities = ({ useStyles }) => {
+  const [city, setCity] = useState([])
+  const [adding, setAdding] = useState(false)
+  const [added, setAdded] = useState(false)
+  const classes = useStyles()
+
+  const handlerCityChange = (e) => {
+    setCity(e.target.value)
+  }
+  const addNext = () => {
+    setCity([])
+    setAdding(false)
+    setAdded(false)
+  }
+  const addCities = async (e) => {
+    e.preventDefault()
+    try {
+      setAdding(true)
+      const addedCity = await fetchData(
+        global.API_BASE_URL + 'api/admin/city',
+        'POST',
+        {
+          name: city,
+        }
+      )
+      if (addedCity) {
```

Comment:
if request passes we will always have value, so this if statement is not necessary